app/scripts/controllers/build.js

```diff
@@ -111,101 +131,139 @@ angular.module('openshiftConsole')
-            .then(function resolve(build) {
+          BuildsService.cancelBuild(
+            $scope.build,
+            $scope.buildConfigName,
+            context
+          ).then(
+            function resolve(build) {
               // TODO: common alerts service to eliminate duplication
               $scope.alerts["cancel"] = {
                 type: "success",
-                message: "Cancelled build " + build.metadata.name + " of " + $scope.buildConfigName + "."
+                message: (
+                  "Cancelled build " +
+                    build.metadata.name +
+                    " of " +
+                    $scope.buildConfigName +
+                    "."
+                )
               };
-            }, function reject(result) {
+            },
+            function reject(result) {
               // TODO: common alerts service to eliminate duplication
               $scope.alerts["cancel"] = {
                 type: "error",
                 message: "An error occurred cancelling the build.",
-                details: $filter('getErrorDetails')(result)
+                details: $filter("getErrorDetails")(result)
               };
-            });
+            }
+          );
         };
 
         var getLinksClonedBuild = function(build) {
           // When the build is first cloned, the Jenkins annotations are not
           // yet updated, so the Jenkins log link is wrong. Give a link to the
           // build instead of the log. Also link to the build if the user
           // doesn't have authority to view the log.
-          if ($filter('isJenkinsPipelineStrategy')($scope.build) ||
-              !$filter('canI')('builds/log', 'get')) {
-            return [{
-              href: Navigate.resourceURL(build),
-              label: "View Build"
-            }];
+          if (
+            $filter("isJenkinsPipelineStrategy")($scope.build) ||
+              !$filter("canI")("builds/log", "get")
+          ) {
+            return [
+              {
+                href: Navigate.resourceURL(build),
+                label: "View Build"
+              }
+            ];
           }
 
-          var logLink = $filter('buildLogURL')(build);
+          var logLink = $filter("buildLogURL")(build);
           if (!logLink) {
             return [];
           }
 
-          return [{
-            href: logLink,
-            label: "View Log"
-          }];
+          return [
+            {
+              href: logLink,
+              label: "View Log"
+            }
+          ];
         };
 
         $scope.cloneBuild = function() {
-          var name = _.get($scope, 'build.metadata.name');
+          var name = _.get($scope, "build.metadata.name");
           if (name && $scope.canBuild) {
-            BuildsService
-              .cloneBuild(name, context)
-              .then(function resolve(build) {
+            BuildsService.cloneBuild(name, context).then(
```

Comment:
Over the years, I've found that the precise style doesn't matter to me so much as consistency between files. I've worked in codebases that used 3-space indents, which seem like some sort of bizarre compromise between 2- and 4-space (and maybe it was), but I got used to it eventually. We could bikeshed all day about the best way to format code, but ultimately what matters is consistency, and that's something tools like `go fmt` do well.

I'd like to suggest that perhaps we (as developers and humans) underestimate our ability to adapt :smile: